### PR TITLE
[new release] shuttle_ssl, shuttle_http and shuttle (0.8.1)

### PR DESCRIPTION
**Pull request by anuragsoni**
- Project page: <a href="https://github.com/anuragsoni/shuttle">https://github.com/anuragsoni/shuttle</a>
- Documentation: <a href="https://anuragsoni.github.io/shuttle/">https://anuragsoni.github.io/shuttle/</a>

##### CHANGES:

* Revive the http codec as a new shuttle_http package
  - Http codec supports a timeout for reading Request headers
  - Server module reports a deferred that resolves when the server context closes. This can be usedul to register cleanup actions that should run when a server connection is torn down.
  - Using the utility methods within the Server module to create responses ensures that streams are torn down if the server connection is closed before a stream was fully consumed.

### CODE REVIEW

File: packages/shuttle_http/shuttle_http.0.8.1/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
Comment (mseri) [inline packages/shuttle_http/shuttle_http.0.8.1/opam:29]:
```suggestion
    "@runtest" {with-test & os-distribution != "macos"}
```
The sandbox does not allow to bind the address for the test

File: packages/shuttle_http/shuttle_http.0.8.1/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test & os-distribution != "macos"}
```
Comment (mseri) [inline packages/shuttle_http/shuttle_http.0.8.1/opam:29]:
```suggestion
    "@runtest" {with-test & os != "macos"}
```

Comment (anuragsoni) [reply packages/shuttle_http/shuttle_http.0.8.1/opam]: Thanks! I'll reproduce this in the opam file in my project upstream for future releases.

**Comment by mseri** [comment]
Thanks

**Pull request merged**